### DISCUSSION TITLE
Recently update

### DISCUSSION THREAD

**Discussion by KainanSu** [requarks/wiki#5652] [Help / Questions] [answered]
Hi!
I wonder if we can see recently update, 
including pages, comments and images, 
without a admin identity?

**Comment by KainanSu** [comment]
We used `confluence` before. 
It show us who update what as a list sort by date. 
I think it is an indispensable feature.
So I would like to know there was similar feature in `wikijs`?

**Comment by NGPixel** [comment, marked as answer]
You can see recently updated pages in the Administration dashboard.

The ability to add a page viewable by normal users will be added in the upcoming 3.0 version.

**Comment by KainanSu** [comment]
@NGPixel Thanks, I expect it so much!
